js/form.js

```diff
@@ -1,47 +1,88 @@
-import { isEscEvent, isFocusElement } from './utils.js';
+import { isEscEvent, isFocusElement, hideElement, showElement } from './utils.js';
 import { validationHashTags } from './validation-hash-tags.js';
 import { validationComments } from './validation-comments.js';
+import { decreaseScale, increaseScale } from './change-scale.js';
+import { changeEffectPhoto, effectInputRadioContainer, removeEffectPhoto } from './change-effect.js';
 
 const uploadPictureForm = document.querySelector('.img-upload__form');
 const editPicturePopup = uploadPictureForm.querySelector('.img-upload__overlay');
 const uploadPictureInput = uploadPictureForm.querySelector('#upload-file');
 const closeEditPictureButton = uploadPictureForm.querySelector('#upload-cancel');
 const hashtagsInput = uploadPictureForm.querySelector('.text__hashtags');
 const commentsInput = uploadPictureForm.querySelector('.text__description');
+const smallerScaleButton = uploadPictureForm.querySelector('.scale__control--smaller');
+const biggerScaleButton = uploadPictureForm.querySelector('.scale__control--bigger');
+const inputScaleValue = uploadPictureForm.querySelector('.scale__control--value');
+
+const START_VALUE_EFFECT = '100%';
 
 const onDocumentKeydown = (evt) => {
   if (isEscEvent(evt) && !(isFocusElement(hashtagsInput)) && !(isFocusElement(commentsInput))) {
     evt.preventDefault();
-    // eslint-disable-next-line no-use-before-define
     closeEditPopup();
   }
 };
 
-// eslint-disable-next-line no-use-before-define
 const onCloseEditPictureButtonClick = () => closeEditPopup();
 
+const onHashtagsInputChange = () => validationHashTags();
+
+const onCommentsInputInput = () => validationComments();
+
+const onSmallerScaleButtonClick = () => decreaseScale();
+
+const onBiggerScaleButtonClick = () => increaseScale();
+
+const onEffectInputRadioContainerChange = (evt) => {
+  if (evt.target.matches('input[type="radio"]')) {
+    changeEffectPhoto(evt);
```

Comment:
```suggestion
    changeEffectPhoto(evt.target.value);
```

https://github.com/htmlacademy-javascript/1416813-kekstagram-23/pull/9#discussion_r673169443